packages/@aws-cdk/aws-ecs/test/test.scheduled-ecs-task.ts

```diff
@@ -0,0 +1,281 @@
+          Links: [],
+          LinuxParameters: {
+            Capabilities: {
+              Add: [],
+              Drop: []
+            },
+            Devices: [],
+            Tmpfs: []
+          },
+          LogConfiguration: {
+            LogDriver: "awslogs",
+            Options: {
+              "awslogs-group": {
+                Ref: "ScheduledEc2TaskScheduledTaskLoggingLogGroupDBEF2BF3"
+              },
+              "awslogs-stream-prefix": "ScheduledEc2Task",
+              "awslogs-region": {
+                Ref: "AWS::Region"
+              }
+            }
+          },
+          Memory: 512,
+          MountPoints: [],
+          Name: "ScheduledContainer",
+          PortMappings: [],
+          Ulimits: [],
+          VolumesFrom: []
+        }
+      ]
+    }));
+
+    test.done();
+  },
+
+  "Scheduled Ec2 Task - with MemoryReservation defined"(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.VpcNetwork(stack, 'Vpc', { maxAZs: 1 });
+    const cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
+    cluster.addCapacity('DefaultAutoScalingGroup', {
+      instanceType: new ec2.InstanceType('t2.micro')
+    });
+
+    new ScheduledEc2Task(stack, 'ScheduledEc2Task', {
+      cluster,
+      image: ecs.ContainerImage.fromRegistry('henk'),
+      memoryReservationMiB: 512,
+      scheduleExpression: 'rate(1 minute)'
+    });
+
+    // THEN
+    expect(stack).to(haveResource('AWS::ECS::TaskDefinition', {
+      ContainerDefinitions: [
+        {
+          Essential: true,
+          Image: "henk",
+          Links: [],
+          LinuxParameters: {
+            Capabilities: {
+              Add: [],
+              Drop: []
+            },
+            Devices: [],
+            Tmpfs: []
+          },
+          LogConfiguration: {
+            LogDriver: "awslogs",
+            Options: {
+              "awslogs-group": {
+                Ref: "ScheduledEc2TaskScheduledTaskLoggingLogGroupDBEF2BF3"
+              },
+              "awslogs-stream-prefix": "ScheduledEc2Task",
+              "awslogs-region": {
+                Ref: "AWS::Region"
+              }
+            }
+          },
+          MemoryReservation: 512,
+          MountPoints: [],
+          Name: "ScheduledContainer",
+          PortMappings: [],
+          Ulimits: [],
+          VolumesFrom: []
+        }
+      ]
+    }));
+
+    test.done();
+  },
+
+  "Scheduled Ec2 Task - with Command defined"(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.VpcNetwork(stack, 'Vpc', { maxAZs: 1 });
+    const cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
+    cluster.addCapacity('DefaultAutoScalingGroup', {
+      instanceType: new ec2.InstanceType('t2.micro')
+    });
+
+    new ScheduledEc2Task(stack, 'ScheduledEc2Task', {
+      cluster,
+      image: ecs.ContainerImage.fromRegistry('henk'),
+      memoryReservationMiB: 512,
+      command: "-c, 4, amazon.com",
```

Comment:
....is this actually a valid command?

Reply:
it's not for this image, but i can update it (was using the command referenced here: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/example_task_definitions.html#example_task_definition-ping), the purpose of this test was really to ensure that the command property was being split on the comma delimiter. L240-L245. I wanted to ensure that in the future, if the cdk.Fn.split() method was modified, this test would catch any breaking changes.

Reply:
Ah I see, makes sense. Just wanted to make sure the test was a realistic example, but this is fine for now.